core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceRepository.scala

```diff
@@ -41,7 +72,43 @@ class WorkspaceRepository(dataSource: SlickDataSource) {
       access.workspaceQuery.delete(workspace.toWorkspaceName)
     }
 
+  def deleteWorkspaceRecord(workspaceName: WorkspaceName): Future[Boolean] =
+    dataSource.inTransaction { access =>
+      access.workspaceQuery.delete(workspaceName)
+    }
+
   def updateCompletedCloneWorkspaceFileTransfer(wsId: UUID, finishTime: DateTime): Future[Int] =
     dataSource.inTransaction(_.workspaceQuery.updateCompletedCloneWorkspaceFileTransfer(wsId, finishTime.toDate))
 
+  def createNewMCWorkspaceRecord(workspaceId: UUID,
```

Comment:
Two things:
* Same naming nit: `createMCWorkspace` (`new` goes without saying I think). 
* I'm wondering if it we should remove the dependency on `WorkspaceRequest`, as that feels like an API layer model leaking into the repo layer, especially since the only two things we care about on the request are teh name and the attributes.